packages/@vue/cli-plugin-e2e-nightwatch/nightwatch.config.js

```diff
@@ -28,6 +28,9 @@ module.exports = deepmerge({
     chrome: {
       desiredCapabilities: {
         browserName: 'chrome',
+        chromeOptions: {
+            w3c: false
```

Comment:
Thanks for the help! But could you please reformat the code to use 2-space indentation here?